.github/workflows/conformance.yml

```diff
@@ -255,6 +270,7 @@ jobs:
         fi
         set -e
 
+        cat cover.out
```

Comment:
Is this needed? May create a lot of noise

Reply:
Removed